Backend/src/Modules/Notes/Notes.Application/Entities/Notes/Queries/GetNoteById/GetNoteByIdHandler.cs

```diff
@@ -7,12 +7,14 @@ internal class GetNoteByIdHandler(INotesDbContext dbContext) : IQueryHandler<Get
 {
     public async Task<GetNoteByIdResult> Handle(GetNoteByIdQuery query, CancellationToken cancellationToken)
     {
+        var noteId = query.Id.ToString();
+
         var note = await dbContext.Notes
             .AsNoTracking()
-            .SingleOrDefaultAsync(n => n.Id == NoteId.Of(Guid.Parse(query.Id)), cancellationToken);
+            .SingleOrDefaultAsync(n => n.Id == NoteId.Of(Guid.Parse(noteId)), cancellationToken);

```

Comment:
...and then `string` to `NoteId` 😏 

```suggestion
            .SingleOrDefaultAsync(n => n.Id == query.Id, cancellationToken);
```